index.bs

```diff
@@ -3968,11 +3951,7 @@ To <dfn>trigger event-level attribution</dfn> given an [=attribution trigger=] |
     <dl class="switch">
     : "<code>[=event-level-report-replacement result/add-new-report=]</code>"
     ::
-         1. Let |numMatchingReports| be the number of entries in the [=event-level report cache=] whose
-             [=event-level report/attribution destinations=] [=set/contains=] |trigger|'s [=attribution trigger/attribution destination=].
-         1. If |numMatchingReports| is greater than or equal to the user agent's [=max event-level reports per attribution destination=]:
-             1. Return the [=triggering result=] ("<code>[=triggering status/dropped=]</code>",
-                 ("<code>[=trigger debug data type/trigger-event-storage-limit=]</code>", null)).
+         1. [=iteration/Break=].
```

Comment:
@apasel422 does break seem the correct action to skip from here to the next step (16)?

Reply:
Actually no, because there is no ongoing loop. I'd just say:

```
1. Do nothing.
```

Reply:
Done!